<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222433;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="QT_btStart">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btQ1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Q1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btQ2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Q2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QT_btKickOff">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
    </property>
    <property name="text">
     <string>Kickoff</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btBlue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QT_btFreeBall">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
    </property>
    <property name="text">
     <string>Free Ball</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QT_btGoalKick">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
    </property>
    <property name="text">
     <string>Goal Kick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QT_btPenaltyKick">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
    </property>
    <property name="text">
     <string>Penalty Kick</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btQ4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Q4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btYellow">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;

</string>
    </property>
    <property name="text">
     <string>Yellow</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btStop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btHalt">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Halt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QT_btQ3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>Q3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btPararTransmissao">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>510</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>24</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
    </property>
    <property name="text">
     <string>Parar transmissão</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btJogar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>4</red>
          <green>99</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #046307;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #41a317;
	border-radius:0px;
}</string>
    </property>
    <property name="text">
     <string>Iniciar transmissão</string>
    </property>
   </widget>
   <widget class="QLabel" name="qt_Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="qt_UltimaFalta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="qt_FaltaAtual">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: #9F1823;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
